types/socket.io/v1/index.d.ts

```diff
@@ -50,7 +50,7 @@ interface SocketIOStatic {
 
 declare namespace SocketIO {
     interface Server {
-        engine: { ws: any };
+        engine: { ws: any, generateId: () => {} };
```

Comment:
same as with current version

Reply:
I'm not very sure what the "v1" folder is for. Do I need to leave this unchanged in my case?